config/ignition_edifice_debian_buster.yaml

```diff
@@ -2,120 +2,148 @@ name: ignition_edifice_debian_buster
 method: http://packages.osrfoundation.org/gazebo/debian-stable
 suites: [buster]
 component: main
-architectures: [amd64, source]
+architectures: [amd64, armhf, arm64, source]
 filter_formula: "\
-(Package (= ignition-edifice)
-), $Version (% 1.0.2-*) |\
+(Package (= ignition-edifice) \
+), $Version (% 1.0.0-1*) |\
 (Package (= ignition-common4) |\
  Package (= libignition-common4) |\
  Package (= libignition-common4-av) |\
+ Package (= libignition-common4-av-dbgsym) |\
  Package (= libignition-common4-av-dev) |\
  Package (= libignition-common4-core-dev) |\
+ Package (= libignition-common4-dbgsym) |\
  Package (= libignition-common4-dev) |\
  Package (= libignition-common4-events) |\
+ Package (= libignition-common4-events-dbgsym) |\
  Package (= libignition-common4-events-dev) |\
  Package (= libignition-common4-graphics) |\
+ Package (= libignition-common4-graphics-dbgsym) |\
  Package (= libignition-common4-graphics-dev) |\
  Package (= libignition-common4-profiler) |\
- Package (= libignition-common4-profiler-dev)
-), $Version (% 4.4.0-*) |\
+ Package (= libignition-common4-profiler-dbgsym) |\
+ Package (= libignition-common4-profiler-dev) \
+), $Version (% 4.5.0-1*) |\
 (Package (= ignition-fuel-tools6) |\
  Package (= libignition-fuel-tools6) |\
- Package (= libignition-fuel-tools6-dev)
-), $Version (% 6.1.0-*) |\
+ Package (= libignition-fuel-tools6-dbgsym) |\
+ Package (= libignition-fuel-tools6-dev) \
+), $Version (% 6.2.0-1*) |\
 (Package (= ignition-gazebo5) |\
  Package (= libignition-gazebo5) |\
+ Package (= libignition-gazebo5-dbg) |\
  Package (= libignition-gazebo5-dev) |\
- Package (= libignition-gazebo5-plugins)
-), $Version (% 5.3.0-*) |\
+ Package (= libignition-gazebo5-plugins) \
+), $Version (% 5.2.0-1*) |\
```

Comment:
Here's another "downgrade" but in this case the imported package is actually still at 5.0.0 so this will affect an upgrade.

```
ignition-gazebo5_5.0.0-1~buster.debian.tar.xz      16-Apr-2021 13:57                2068
ignition-gazebo5_5.0.0-1~buster.dsc                16-Apr-2021 13:57                1858
libignition-gazebo5-dev_5.0.0-1~buster_amd64.deb   16-Apr-2021 13:57              137996
libignition-gazebo5-plugins_5.0.0-1~buster_amd6..> 16-Apr-2021 13:57             3013128
libignition-gazebo5_5.0.0-1~buster_amd64.deb       16-Apr-2021 13:57             3906496
```